src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs

```diff
@@ -133,14 +132,21 @@ public void InitializeService(ServiceHost serviceHost)
         {
             try
             {
+                ConnectionInfo connInfo;
+                ScriptingService.ConnectionServiceInstance.TryFindConnection(
+                    parameters.ConnectionString,
```

Comment:
connection string has a specific format that is well know.  this property should not contain input that doesn't follow that format.  if we need to pass a owneruri please define an option owneruri parameter

---
In reply to: [141198831](https://github.com/Microsoft/sqltoolsservice/pull/446#discussion_r141198831) [](ancestors = 141198831)

Reply:
also, since we are changing method formats for JSON-RPC DMP, please verify these changes work with mssql-scripter

---
In reply to: [141472723](https://github.com/Microsoft/sqltoolsservice/pull/446#discussion_r141472723) [](ancestors = 141472723,141198831)